requirements_spark.txt

```diff
@@ -5,7 +5,6 @@ pika==0.13.0
 python-dateutil==2.8.0
 # Numpy has dropped support for Python 3.6.x from version 1.20.0. So we use version 1.19.5 till we upgrade Python version in Spark images.
 numpy==1.19.5
-git+https://github.com/metabrainz/brainzutils-python.git@v1.16.1
```

Comment:
if we're going to remove this should we also remove flask, werkzeug, etc, etc?

Reply:
my bad :sweat: